pkgs/tools/system/nsc/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, buildGoModule
```

```suggestion
, stdenv
, buildGoModule
```

Because it is broken on Darwin.

---

:+1: might try to get it working on Darwin later, but this sounds good for now!